include/deal.II/base/symmetric_tensor.h

```diff
@@ -1217,14 +1217,14 @@ namespace internal
       case 2:
         return (data[0] * sdata[0] +
                 data[1] * sdata[1] +
-                Number(2.) * data[2] * sdata[2]);
```

Comment:
i am afraid that this may break some tests with `std::complex<>` in `tests/base/`, let us know how does it look.